pkg/resources/user_parameters_test.go

```diff
@@ -0,0 +1,14 @@
+package resources
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_UserParametersSchema(t *testing.T) {
+	t.Run("description references parameter docs correctly", func(t *testing.T) {
+		require.True(t, strings.HasSuffix(userParametersSchema["abort_detached_query"].Description, "For more information, check [ABORT_DETACHED_QUERY docs](https://docs.snowflake.com/en/sql-reference/parameters#abort-detached-query)."))
```

Comment:
Do we really need that? It's almost the same like pkg/resources/resource_parameters_commons_test.go

Reply:
Later we won't, but currently these tests are different:
- this one validates that the info about param docs is added
- the second one you mention validates ho the message is constructed.

You are right they are pretty similar and the second one could be removed. They don't hurt us much now and I would like to have later a generic solution transforming the SDK definition into tf schema, then we will probably change this test a bit.

WDYT?